main/options/options.cc

```diff
@@ -736,8 +736,8 @@ bool extractPrinters(cxxopts::ParseResult &raw, Options &opts, shared_ptr<spdlog
     if (raw.count("print") == 0) {
         return true;
     }
-    vector<string> printOpts = raw["print"].as<vector<string>>();
-    for (auto opt : printOpts) {
+    const vector<string> &printOpts = raw["print"].as<vector<string>>();
+    for (string_view opt : printOpts) {
```

Comment:
Would it make sense to inline `printOpts` here and make the argument `const auto &opt`? That would be more consistent with the way we're handling options in the rest of this function.

Reply:
We could inline `printOpts` no problem.

Using `const auto &opt` means that we have to rewrite parts of this function, as we modify `opt` just a few lines after this.  `string_view` sneakily works around this, though it's a little unorthodox as the iteration type here.

Reply:
The `string_view` arg seems like a good change, and I'm fine with merging as-is if you don't want to also inline `printOpts` 👍